### DISCUSSION TITLE
qnap, trying to install docker, struggling with the vpn section ( i have airvpn)

### DISCUSSION THREAD

**Discussion by kyrusdemnati** [haugene/docker-transmission-openvpn#2173] [Q&A]
i have run the container via the qnap container station menu but i am missing the openvpn fies, so it can't run
i dont understand ssh etc, if someone explains to me what i need to do, i can probably do it..

i have downloaded the single .opvn from airvpn website and saved in the contrainer / transmission/config directory  as default.opvn

in the contrainer settings i do note the following;

OPENVPN_PASSWORD **None**
OPENVPN_PASSWORD **None**
OPENVPN_USERNAME **None**

i have tried googling it and finding tutorials, but i can't get my head around it...

"***
Exiting...
Starting container with revision: 20877f1b168b6ff27fc58aeef40756572c562d47
Creating TUN device /dev/net/tun
ERROR: Cannot determine where to find your OpenVPN config. Both OPENVPN_CONFIG_URL and OPENVPN_PROVIDER is unset.
You have to either provide a URL to the config you want to use, or set a configured provider that will download one for you.

**Comment by pkishino** [comment]
you could try using portainer to manage your containers.. you need to set the parameters, otherwise it won't work.
But just a heads up.. qnap is notoriously bad for this as it actively tries to kill any openvpn process. Others here have run into this and if you google you should find results on this. So I recommend using a different platform.
The other parts are mostly answered in the documentation for this container.
